src/domain/notifications/index.types.d.ts

```diff
@@ -18,14 +18,14 @@ type OnChainTxPaymentArgs = OnChainTxBaseArgs
 type LnInvoicePaidBitcoinWalletArgs = {
   paymentHash: PaymentHash
   recipientWalletId: WalletId
-  sats: Satoshis
+  sats: Satoshis | bigint
```

Comment:
shouldn't this be `PaymentAmount<T>`?

Reply:
This is one of the places where I started to do the conversion but then it snowballed through the other transactionNotification functions within this so I abandoned for a later PR